src/components/Charts/Axis.tsx

```diff
@@ -1,10 +1,18 @@
 import React from 'react';
-import { timeWeek } from 'd3-time';
+import { timeDay } from 'd3-time';
```

Comment:
Changed it to use days to make sure that the first tick matches the first day in the range, when using `timeWeek` it was rounding up to the first Sunday after the first date.